content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
+- **Added** support for populating AWS platform metadata in Envoy node metadata.
+- **Improved** Stackdriver adapter for Mixer to support configurable flush intervals for tracing data.
+- **Added** support for a headless collector service to the Jaeger addon.
+- **Fixed** `kubernetesenv` adapter to provide proper support for pods that contain a dot in their name.
+- **Improved** the Fluentd adapter for Mixer to provide millisecond-resolution in exported timestamps.
+
+
+## Configuration management
+
+## Operator
+
+- **Improved** the alpha `IstioControlPlane` API by replacing with the `IstioOperator` API.
+**Added** `istioctl operator init` and `istioctl operator remove` commands istio/operator#643.
+**Improved** reconciliation speed with caching #20344.
+
+## `area/perf and scalability`
+
+- **Improved** Skip unused services while generate clusters for the gateways #20124.
+- **Improved** Skip calling `updateEDS` for headless service #18952.
+- disable `SNI-DNAT` at ingress gateway by default #18431.
+- The declaration of err overrides #19473.
+- create slice with capacity, when capacity is known #18504.
+
+## `area/test and release`
+
+- **Added** Publish a `docker/istioctl` image #19079.
+
+## `istioctl`
+- **Added** mTLS analyzer - #18350
+- **Added** JwtAnalyzer - #20812
+- **Added** ServiceAssociationAnalyzer #19383
+- **Added** SercretAnalyaer #19583
+- **Added** sidecar ImageAnalyzer  #20929
+- **Added** PortNameAnalyzer #19375
+- **Added** Policy DeprecatedAnalyzer #20919
+- **Updated** more validation rules for `RequestAuthentication` #19369.
+- **Added** a new flag `-A|--all-namespaces` to `istioctl analyze` to analyze the entire cluster #19209.
+- **Added** support for analyzing content passed via `stdin` to `istioctl analyze` #19393.
+- **Added** `istioctl analyze -L` to show a list of all analyzers available #19200.
+- **Added** the ability to suppress messages from `istioctl analyze` #19673.
+- **Added** structured format options to `istioctl analyze` #18700.
+- **Added** links to relevant documentation to `istioctl analyze` output #19105.
+- **Updated** annotation methods provided by Istio API in analyzer #18829.
+- **Updated** istioctl analyze now loads files from a directory #20718
+- **Updated** istioctl analyze to try to associate message with their source filename
+- **Updated** istioctl analyze to print the namespace that is being analyzed #20515
+- **Updated** istioctl analyze to analyze in-cluster resources by default. #19647
+- **Fixed** bug where analyzer suppressed cluster-level resource messages. #18935.
+- **Added** support for multiple input files to `istioctl manifest` #20190.
+- **Replaced** the `IstioControlPlane` API with the `IstioOperator` API.
+- **Added** selector for `istioctl` dashboard #19191.
+- **Added** support for slices and lists in `istioctl manifest --set` flag #20631.\
+- **Graduated** `istioctl analyze` out of experimental  #19488.
```

Comment:
We need to add a note in release note that multicluster is broken and if you are using multicluster you should not update to 1.5

Reply:
we also should put in a summary for upgrade, and refer to upgrade notice.